data_sink_client.rb

```diff
@@ -0,0 +1,20 @@
+require 'oj'
+
+class DataSinkClient
+  include Singleton
```

Comment:
Why not class methods?

Reply:
I think this way is more concise for this particular usecase where I do not want to instance a client multiple times